data/migrations/20211215202038_Submissions_Table.js

```diff
@@ -0,0 +1,19 @@
+exports.up = function (knex) {
+  return knex.schema
+    .alterTable('Submissions-New', (t) => {
+      t.timestamp('startAt');
+      t.timestamp('readAt');
+      t.timestamp('drawAt');
+      t.timestamp('writeAt');
+      t.timestamp('squadUpAt');
+      t.timestamp('voteAt');
+
+      t.string('gameMode').notNullable();
+    })
+    .dropTable('Submissions-New')
+    .renameTable('Submissions-New', 'Submissions');
```

Comment:
I don't think you need to drop the table if you are renaming it.

Reply:
Thanks. I've removed this and just did the alteration until we can figure out how to rename and drop the old table.